test-utils/src/lib.rs

```diff
@@ -165,3 +165,37 @@ pub fn drop_catalog_data<T: Into<String>>(
     }
     Ok(())
 }
+
+/// check_server_version_for_test verifies that the server version is valid if the test specifies min or max server versions.
+fn check_server_version_for_test(
+    min_server_version: &Option<String>,
+    max_server_version: &Option<String>,
+    server_version: &Option<String>,
+) -> bool {
+    // max server version implies there is some version above which the test should not run; hence
+    // if the server version is "latest" we skip the test.
+    if server_version == &Some("latest".to_string()) {
+        return !max_server_version.is_some();
+    }
+    // rapid can be a moving target, so for now, we will skip tests that have version constraints on rapid
```

Comment:
as mentioned in this comment, rapid right now is 8.2.1; this means that for now, it disagrees with "latest" when it comes to how these specific tests behave. And in general, these bleeding edge versions probably will waffle back and forth and require a lot of small changes as rapid catches up with latest. So for now, I just skip tests for `rapid` if a min or max server version is specified.